src/dspaces-server.c

```diff
@@ -1216,6 +1221,8 @@ static int server_destroy(dspaces_provider_t server)
     }
 
     free_sspace(server->dsg);
+    remove_dir_rf(server->conf.swap.file_dir);
```

Comment:
ah, interesting point about using `"."`. I have to check this special case and use regex to remove all the swap files. But basically, the philosophy here is: the swap file should be only valid during a single run.

Reply:
I think it's reasonable to ask the user to cleanup their own swap files. I hate deleting files in a code; it can be very dangerous and creates security risks.